src/customButton.svelte

```diff
@@ -0,0 +1,24 @@
+<script>
+    export let action=()=>null;
+</script>
+
+<button class="cta" on:click={action} >
+    Custom button &#129409
```

Comment:
why is this the label of the custom button?

Reply:
Sorry I was quite a bit in a rush back there! Do you have a better label for it?